.circleci/config.yml

```diff
@@ -463,6 +502,17 @@ workflows:
             - AWS__PHXDEVOPS__circle-ci-test
             - GCP__automated-tests
             - GITHUB__PAT__gruntwork-ci
+      - test_signing:
+          requires:
+            - build
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
No, it'll run whenever the build is done (it has the same filter as build):
https://app.circleci.com/pipelines/github/gruntwork-io/terragrunt/3824/workflows/0ad3d122-0ea4-490e-939e-5be7917c0ced

We might want to not run it for releases, as it doesn't really matter if we test it at that stage, as we''ll be signing it for real. I left it in because I thought it wouldn't hurt, but I cant remove it.